tests/fuzz/dictionary_round_trip.c

```diff
@@ -42,21 +43,33 @@ static size_t roundTripTest(void *result, size_t resultCapacity,
                 dict.buff, dict.size,
                 cLevel);
     } else {
-        dictContentType = FUZZ_dataProducer_uint32Range(producer, 0, 2);
+        /* refPrefix seems to fail when not used with auto? */
+        if (!refPrefix)
+            dictContentType = FUZZ_dataProducer_uint32Range(producer, 0, 2);
```

Comment:
Gives me dictionary mismatch error when I try to use refPrefix with full_dict. Not sure if this is a real bug or if I'm doing something wrong.